### ISSUE TITLE
False error: pylint + pytest 6 - pytest.mark.parametrize is not callable (not-callable)

### ISSUE THREAD

**Issue by adamtheturtle** [PyCQA/pylint#3755]
### Steps to reproduce

Use [`pytest.mark.parametrize`](https://docs.pytest.org/en/stable/parametrize.html), e.g.

```python
"""
Example pytest mark.
"""

import pytest

@pytest.mark.parametrize('sample', [0])
def test_example(sample):
    """
    Example test.
    """
    assert sample == 0

```

Install `pytest==6.0.0`.
Run `pylint`.

### Current behavior

* The test passes when running `pytest`.

I see the error:

```
test_example.py:7:1: E1102: pytest.mark.parametrize is not callable (not-callable)
```

This error is not shown with `pytest==5.4.3`.

### Expected behavior

No error is shown.

### pylint --version output

```
pylint 2.5.3
astroid 2.4.2
Python 3.8.5 (default, Jul 21 2020, 10:48:26) 
[Clang 11.0.3 (clang-1103.0.32.62)]
```

**Comment by k0pak4** [comment]
If it helps any, I am seeing the same errors with `pytest.mark.skip`, `pytest.mark.skipif`, and `pytest.mark.xfail` on version:
`pylint 2.5.3
astroid 2.4.2
Python 3.7.4 (default, Aug 14 2019, 12:19:22)
[GCC 8.3.0]`
and pytest 6.0.0

**Comment by adamtheturtle** [comment]
It seems that the `pytest` folks are handling this particular case - https://github.com/pytest-dev/pytest/pull/7565.

**Comment by adamtheturtle** [comment]
`pytest` has now released a change which avoids this error. I guess either this issue should be closed, or it should be investigated why `pylint` showed an error on Python code which worked.

**Comment by PCManticore** [comment]
Thanks for the report. Since this is fixed on `pytest` side, let's close this. We can definitely investigate why this was the case, but with 600+ opened issues, I doubt it's that important.
